lib/getReadmes.js

```diff
@@ -0,0 +1,74 @@
+var https = require('https');
+var url = require('url');
+var env = require('env2')('./config.env');
```

Comment:
@des-des if you are using a `.env` file would you mind including a sample in the `README.md` so others who want to contribute to this project know where to start?